### DISCUSSION TITLE
Error when applying IP filter to Sigma rules.

### DISCUSSION THREAD

**Discussion by Johnny111125** [Security-Onion-Solutions/securityonion#13527] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

200 gb

### Storage for /nsm

150 gb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am trying to filter out my security scanner from triggering certain Sigma rules, however it seems the filter is breaking the rule. After applying the filter, the alert will not fire at all. I am also receiving an error when testing the filter in Kibana. 
Am I applying this correctly?
![image](https://github.com/user-attachments/assets/7644be28-d470-454c-90f2-7fc2320ced99)

Error when testing in Kibana.
![image](https://github.com/user-attachments/assets/1f2fce3f-2ff6-46a0-8a8b-3234bbf23a77)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
Can you share one of the actual alerts, so we can see what fields are included?

**Comment by Johnny111125** [comment]
This is one of the alerts. I would like to block 192.168.1.223 from triggering this.

Security Onion IDH - SSH Accessed

![image](https://github.com/user-attachments/assets/f7cc34d3-bd0c-4d11-8b18-f11b1b9c98a4)

**Reply by InfosecGoon** [reply]
Does a filter of "source.ip:192.168.1.223" do it?

It looks like the issue is that your current filter is trying to run a CIDR function on a non-IP field, so the query isn't being understood.

**Reply by Johnny111125** [reply]
Yes, that works. Thanks!

Now, how do I filter multiple IPs?

**Reply by Johnny111125** [reply]
I thought by just adding a second filter I would be able to filter another IP, however that did not work. What is the proper way to filter multiple IPs?
